crawler/__init__.py

```diff
@@ -22,20 +22,26 @@ def create_app(config_object: str = None) -> Flask:
     # setup logging
     logging.config.dictConfig(app.config["LOGGING"])
 
-    if app.config.get("ENABLE_CHERRYPICKER_ENDPOINTS", False):
-        from crawler.blueprints import cherrypicker_test_data
-
-        app.register_blueprint(cherrypicker_test_data.bp)
-
     if app.config.get("SCHEDULER_RUN", False):
         scheduler.init_app(app)
         scheduler.start()
 
+    setup_blueprints(app)
+
     @app.get("/health")
-    def health_check():
+    def _():
```

Comment:
I know the route indicates the purpose but maybe elaborating in the function's docstring since it's now only `_`.